### DISCUSSION TITLE
How to send Unicode characters with custom key presses

### DISCUSSION THREAD

**Discussion by davidtoop** [blinksh/blink#1860] [Support]
I am trying to setup an APL keyboard on my iPad and want to send Unicode characters as a custom key press for instance typing option+I should send the APL characters iota U2373. If I set the custom key press to the Unicode I get two characters I(#s) if I try the utf8 encoding I get 3 characters.  Can’t seem to find anything in the documentation.

**Comment by carloscabanero** [comment]
Hi! I think you do not need custom key presses for this. You can configure Option with the extended APL characters by going to config > Keyboard > Option and setting the modifier from Esc to Default. Let me know if that is what you are looking for.

**Reply by davidtoop** [reply]
Sorry that doesn’t work I don’t get the APL characters, for option i I get ^, note I can paste the iota character into the APL session so the problem is not one of font. FYI if I mosh from my Mac then everything works as expected.

**Reply by yury** [reply]
Yep, confirming... moving back to issue

**Comment by yury** [comment]
No way to convert it back :(... Will create new one
